### PR TITLE
Styling issue resolved

### PR DESCRIPTION
**Pull request by vidyaaKhandekar**
## Release Notes

- **New Features**
  - Enhanced routing and loading state management in the main application.
  - Dynamic loading of components for improved performance in the authentication routes.
  - Introduction of a custom hook for tracking device size.

- **Improvements**
  - Improved user interface with clearer instructions and interactive elements in various components.
  - Streamlined layout and structure for better user experience.
  - Updated user details presentation and status representation in the ApplicationList component.
  - Simplified loading message in the Loader component.
  - Enhanced handling of user data and improved robustness in the UserProfile component.

- **Bug Fixes**
  - Resolved issues with user data handling and display logic in the UserProfile component.

- **Style Changes**
  - Adjusted styles for various components to enhance visual consistency and layout.

- **Chores**
  - Removed unused components and streamlined imports for better code maintenance.
  - Updated localization entries for enhanced user experience.

### CODE REVIEW

File: src/components/common/layout/BottomBar.tsx [hunk 1]
```diff
@@ -18,26 +24,23 @@ const BottomBar: React.FC = () => {
...
-          onClick={() => handleNavigation("/home")}
-        >
+      <HStack
+        bg="primary.50"
+        alignItems="center"
+        borderTopWidth={"1px"}
+        borderTopColor={"back"}
+        shadow={"FooterShadow"}
+        justifyContent={"space-around"}
+      >
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/components/common/layout/BottomBar.tsx:42]: @vidyaaKhandekar fix this

File: src/components/common/layout/BottomBar.tsx [hunk 2]
```diff
@@ -18,26 +24,23 @@ const BottomBar: React.FC = () => {
...
-      maxW="550px"
-      height="85px"
-      paddingBottom="8"
+      width={width}
+      bg={"white"}
+      position="fixed"
+      bottom="0"
+      ref={(e) => typeof setRef === "function" && setRef(e)}
+      {...props}
+      zIndex={10}
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/components/common/layout/BottomBar.tsx:33]: @vidyaaKhandekar fix this

File: src/components/common/layout/Layout.tsx [hunk 3]
```diff
@@ -44,57 +46,57 @@ const Layout: React.FC<LayoutProps> = ({
...
-  // if (loading) {
-  //   return (
-  //     <Box display="flex" justifyContent="center" alignItems="center">
-  //       <Spinner size="lg" />
-  //     </Box>
-  //   );
-  // }
+  useEffect(() => {
+    setBodyHeight(height - (96 + (navHeader?.current?.clientHeight || 0)));
+  }, [height, navHeader?.current?.clientHeight]);
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/components/common/layout/Layout.tsx:56]: @vidyaaKhandekar fix this

File: src/components/common/layout/useDeviceSize.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+function useDeviceSize({
+  maxWidth = 550,
+  maxHeight,
+}: UseDeviceSizeProps = {}): DeviceSize {
+  const [size, setSize] = useState<DeviceSize>({
+    width: 0,
+    height: 0,
+  });
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/components/common/layout/useDeviceSize.tsx:20]: @vidyaaKhandekar fix this

File: src/components/common/layout/useDeviceSize.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+          ? Math.min(window.innerHeight, maxHeight)
+          : window.innerHeight,
+      });
+    };
+
+    window.addEventListener("resize", handleResize);
+    handleResize(); // initial call to set size
+
+    return () => window.removeEventListener("resize", handleResize);
+  }, [maxWidth, maxHeight]);
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/components/common/layout/useDeviceSize.tsx:38]: @vidyaaKhandekar fix this

File: src/utils/context/checkToken.tsx [hunk 6]
```diff
@@ -28,14 +28,15 @@ interface UserDocument {
   doc_verified: boolean;
   uploaded_at: string;
   is_uploaded: boolean;
+  length?: number;
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/utils/context/checkToken.tsx:31]: @vidyaaKhandekar fix this

File: src/screens/Home.tsx [hunk 7]
```diff
@@ -59,11 +58,10 @@ const Home: React.FC = () => {
       _heading={{
         beneficiary: true,
         heading: `${userData?.firstName || ""} ${userData?.lastName || ""}`,
-        subHeading: t("PROFILE_LOGGED_IN_WITH_E_Wallet"),
         label: keycloak.tokenParsed?.preferred_username,
       }}
     >
-      <Box shadow="md" borderWidth="1px" borderRadius="md" p={3}>
+      <Box shadow="md" borderWidth="1px" borderRadius="md" p={2}>
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/screens/Home.tsx:64]: @vidyaaKhandekar fix this

File: src/components/common/UploadDocumentEwallet.tsx [hunk 8]
```diff
@@ -63,9 +63,11 @@ const UploadDocumentEwallet = ({ userId }) => {
 
   return (
     <div>
-      <OutlineButton
-        label="Upload Missing Document"
+      <CommonButton
         onClick={sendMessageToIframe}
+        label="Upload Missing Document"
```
Comment (manojLondhe) [inline src/components/common/UploadDocumentEwallet.tsx:68]: use lang string here

File: src/components/common/Loader.tsx [hunk 9]
```diff
@@ -1,12 +1,10 @@
 import { Spinner, Text, VStack } from "@chakra-ui/react";
-import { useTranslation } from "react-i18next";
 
 const Loader = () => {
-  const { t } = useTranslation();
   return (
     <VStack>
       <Spinner color="var(--theme-color)" />
-      <Text color="var(--theme-color)">{t("GENERAL_LOADING")}</Text>
+      <Text color="var(--theme-color)"> Loding...</Text>
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply src/components/common/Loader.tsx:7]: @vidyaaKhandekar fix this

[the remark this answers is not in the document]
Comment (vidyaaKhandekar) [reply src/components/common/Loader.tsx:7]: Resolve

File: src/components/common/layout/BottomBar.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (vidyaaKhandekar) [reply src/components/common/layout/BottomBar.tsx:42]: Resolved

File: src/components/common/layout/Layout.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (vidyaaKhandekar) [reply src/components/common/layout/Layout.tsx:56]: Resolved

File: src/utils/context/checkToken.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (vidyaaKhandekar) [reply src/utils/context/checkToken.tsx:31]: Fixed

**Pull request merged**